core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/dto/ScriptDto.java

```diff
@@ -19,6 +20,7 @@
 @EqualsAndHashCode(callSuper = false)
 @AllArgsConstructor
 @NoArgsConstructor
+@Relation(value = "script", collectionRelation = "scripts")
```

Comment:
This is the way to modify the name of the collection below the _embedded property